src/components/GoalInfo/GoalInfo.tsx

```diff
@@ -0,0 +1,177 @@
+import { Tag, TagCleanButton, nullable } from '@taskany/bricks';
+import { IconArrowRightOutline, IconBinOutline, IconEditOutline, IconPlusCircleOutline } from '@taskany/icons';
+import styled from 'styled-components';
+import { ComponentProps, FC, MouseEvent } from 'react';
+
+import { IssueMeta } from '../IssueMeta';
+import { UserBadge } from '../UserBadge';
+import { UserComboBox } from '../UserComboBox';
+import { TagComboBox } from '../TagComboBox';
+import { GoalParentComboBox } from '../GoalParentComboBox';
+import { ModalEvent, dispatchModalEvent } from '../../utils/dispatchModal';
+import { InlineTrigger } from '../InlineTrigger';
+import { GoalByIdReturnType } from '../../../trpc/inferredTypes';
+
+import { tr } from './GoalInfo.i18n';
+
+const StyledInlineTrigger = styled(InlineTrigger)`
+    margin-left: 5px; // 24 / 2 - 7 center of UserPic and center of PlusIcon
+`;
+
+const StyledInlineInput = styled.div`
+    display: flex;
+    align-items: center;
+    height: 28px;
+`;
+
+const StyledIconEditOutline = styled(IconEditOutline)``;
+
+const StyledInlineUserInput = styled.div`
+    display: flex;
+    align-items: center;
+    height: 28px;
+
+    ${StyledIconEditOutline} {
+        display: none;
+    }
+
+    &:hover ${StyledIconEditOutline} {
+        display: block;
+    }
+`;
+
+const tagsLimit = 5;
+
+type Tag = NonNullable<GoalByIdReturnType>['tags'][number];
+type UserComboBoxProps = ComponentProps<typeof UserComboBox>;
+
+type GoalInfoProps = {
+    goal: NonNullable<GoalByIdReturnType>;
+    participants: string[];
+    onGoalAssigneeChange: UserComboBoxProps['onChange'];
+    onGoalParticipantRemove: (id?: string | null) => ComponentProps<typeof UserBadge>['onCleanButtonClick'];
+    onGoalParticipantAdd: UserComboBoxProps['onChange'];
+    onGoalTagRemove: (tag: Tag) => (e: MouseEvent<Element>) => void;
+    onGoalTagAdd: ComponentProps<typeof TagComboBox>['onChange'];
+    onGoalTransfer: ComponentProps<typeof GoalParentComboBox>['onChange'];
+};
+
+export const GoalInfo: FC<GoalInfoProps> = ({
+    goal,
+    participants,
+    onGoalAssigneeChange,
+    onGoalParticipantRemove,
+    onGoalParticipantAdd,
+    onGoalTagRemove,
+    onGoalTagAdd,
+    onGoalTransfer,
+}) => {
+    return (
+        <div>
```

Comment:
Does this is the part of GoalPage layout? Like this https://github.com/taskany-inc/issues/pull/1745/files#diff-b842f4ad5ef9f56007262c1d647b0bdb1d7bbae4275dda8b2f2ab31f81e7c869R257